oleesir/Banking-app
# #164724559Users should be able to see the create account page

#### What does this PR do?
implement create-account-page

#### Description of Task to be completed?
implement create-account-page

#### How should this be manually tested?
* Navigate to the UI folder and click on create_account.html in your browser
* You will see create_accounts-template

#### What are the relevant pivotal tracker stories?
[164724559](https://www.pivotaltracker.com/story/show/164724559)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-26 at 8 29 29 PM" src="https://user-images.githubusercontent.com/17679511/55027783-6077ca80-5006-11e9-95f8-72c30b47ebbd.png">
